examples/tcpsynbl.yaml

```diff
@@ -0,0 +1,30 @@
+programs:
+  - name: tcpsynbl
+    metrics:
+      histograms:
+        - name: tcp_syn_backlog
+          help: trace the TCP SYN backlog size
+          table: bucket
+          bucket_type: linear
+          bucket_min: 0
+          bucket_max: 1024 # `sysctl net.ipv4.tcp_max_syn_backlog`
+          bucket_multiplier: 1
+          labels:
+            - name: bucket
+              size: 8
+              decoders:
+                - name: uint
+    kprobes:
+      tcp_v4_syn_recv_sock: do_count
+      tcp_v6_syn_recv_sock: do_count
+    code: |
+      #include <net/sock.h>
```

Comment:
📝 `<net/sock.h>` is required to use [`struct sock`](https://github.com/torvalds/linux/blob/caffb99b6929f41a69edbb5aef3a359bf45f3315/include/net/sock.h#L346-L522) and reference to the [`sk_ack_backlog`](https://github.com/torvalds/linux/blob/caffb99b6929f41a69edbb5aef3a359bf45f3315/include/net/sock.h#L478) value later.